poker.test.js

```diff
@@ -0,0 +1,160 @@
+import { expect, it } from "@jest/globals";
+
+const {
+  suitsCount,
+  cardCount,
+  getCardValues,
+  getCardEquivalent,
+  sortCards,
+  checkCardsIfConsecutiveOrder,
+} = require("./poker");
+
+describe("Suit occurrences check", () => {
+  
+  it("Calculate number of suit occurrences if suits are exactly the same", () => {
+    expect(
+      suitsCount([
+        { value: "T", suit: "D" },
+        { value: "J", suit: "D" },
+        { value: "Q", suit: "D" },
+        { value: "K", suit: "D" },
+        { value: "A", suit: "D" },
+      ])
+    ).toMatchObject({ D: 5 });
+  });
```

Comment:
You should create a var called input and var called output for each test:
```js
  it("Calculate number of suit occurrences if suits are exactly the same", () => {
    const input = [
        { value: "T", suit: "D" },
        { value: "J", suit: "D" },
        { value: "Q", suit: "D" },
        { value: "K", suit: "D" },
        { value: "A", suit: "D" },
      ];
      
      const output = { D: 5 }
  
      expect(suitsCount(input)).toMatchObject(output);
  });
```

You should also be using const values for suits and values:
```js
const input = [
    { value: listOfCards.T, suit: suits.Diamonds },
    { value: listOfCards.J, suits.Diamonds },
    { value: listOfCards.Q, suits.Diamonds },
    { value: listOfCards.K, suits.Diamonds },
    { value: listOfCards.A, suits.Diamonds },
];
```

Reply:
Same for other tests